resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/LocalDirsFeatureStep.scala

```diff
@@ -33,12 +34,11 @@ private[spark] class LocalDirsFeatureStep(
   private val useLocalDirTmpFs = conf.get(KUBERNETES_LOCAL_DIRS_TMPFS)
 
   override def configurePod(pod: SparkPod): SparkPod = {
-    var localDirs = pod.container.getVolumeMounts.asScala
+    var localDirs = randomize(pod.container.getVolumeMounts.asScala
       .filter(_.getName.startsWith("spark-local-dir-"))
-      .map(_.getMountPath)
-    var localDirVolumes : Seq[Volume] = Seq()
-    var localDirVolumeMounts : Seq[VolumeMount] = Seq()
-
```

Comment:
Please revert this removal.

Reply:
These line are not removed its there on line 40 and 41 .

Reply:
What I pointed was the removal of that empty line after `var localDirVolumeMounts`.

<img width="1202" alt="Screen Shot 2022-08-20 at 12 45 14 PM" 
src="https://user-images.githubusercontent.com/9700541/185763765-0be7e2df-b13d-4ecf-83b2-f4af6079e408.png">